packages/collector/src/agentConnection.js

```diff
@@ -486,3 +488,26 @@ function logLargeSpans(spans) {
     )}`
   );
 }
+
+/**
+ *
+ * @param {Array.<InstanaBaseSpan>} spans
+ */
+function logSpanDebugInfo(spans) {
+  const spanMapping = {
+    1: 'entrySpans',
+    2: 'exitSpans'
+  };
+
+  const countBySpanType = spans.reduce((acc, item) => {
+    // @ts-ignore
+    const label = spanMapping[item.k];
+    if (label) {
+      // @ts-ignore
+      acc[label] = (acc[label] || 0) + 1;
+    }
+    return acc;
+  }, {});
+
+  logger.debug('error occurred in sendSpans. no of spans found: %s', countBySpanType);
```

Comment:
IMO, it seems more appropriate to categorize this as a `warn` rather than a debug.

Reply:
I agree. Will update